integration/nwo/template/configtx_template.go

```diff
@@ -144,11 +144,19 @@ Profiles:{{ range .Profiles }}
       Addresses:{{ range .Orderers }}{{ with $w.Orderer . }}
       - 127.0.0.1:{{ $w.OrdererPort . "Listen" }}
       {{- end }}{{ end }}
+      {{- if .Blocks}}
```

Comment:
I ran the test and  BatchTimeout and related overrides aren't getting applied. I then did a `spew.Dump(config)` to look at the runtime config:

```  (*nwo.Profile)(0xc00017c3f0)({
   Name: (string) (len=16) "ThreeOrgsChannel",
   Orderers: ([]string) <nil>,
   Consortium: (string) (len=16) "SampleConsortium",
   Organizations: ([]string) (len=3 cap=3) {
    (string) (len=4) "Org1",
    (string) (len=4) "Org2",
    (string) (len=4) "Org3"
   },
   AppCapabilities: ([]string) <nil>,
   ChannelCapabilities: ([]string) <nil>,
   Blocks: (*nwo.Blocks)(0xc000500360)({
    BatchTimeout: (int) 30,
    MaxMessageCount: (int) 30,
    AbsoluteMaxBytes: (int) 1048576,
    PreferredMaxBytes: (int) 1048576
   })
  })```

It appears there are no "Orderers" overridden in this config. And since the `{{- if .Blocks}}` statement is within the `{{- if .Orderers }}` statement, the BatchTimeout and related configs won't get overridden. At least, I think that's why given my limited knowledge of how the templating works.